extensions/query-store/package.json

```diff
@@ -39,9 +39,20 @@
       }
     ],
     "menus": {
-      "objectExplorer/item/context": [],
+      "objectExplorer/item/context": [
+        {
+          "command": "queryStore.openQueryStoreDashboard",
+          "when": "connectionProvider == MSSQL && nodeType == Database  && !(nodePath =~ /^.*\\/System Databases\\/.*$/) && config.workbench.enablePreviewFeatures",
```

Comment:
Will this method work for different languages?  Maybe there's an `.isSystemDatabase` property that this can key off of instead.